app/models/audition.rb

```diff
@@ -1,3 +1,7 @@
 class Audition < ActiveRecord::Base
-  
-end
\ No newline at end of file
+  belongs_to :role
+
+  def call_back
+    self.hired = true
+  end
```

Comment:
This won't save to the DB. You would need to call `self.save` after this or use `update` instead.